test/python/opflow/test_op_construction.py

```diff
@@ -63,6 +63,23 @@ def test_composed_eval(self):
         """ Test eval of ComposedOp """
         self.assertAlmostEqual(Minus.eval('1'), -.5 ** .5)
 
```

Comment:
Could you wrap this in it's own test? That way it's easier to see what breaks, if something breaks 🙂